src/commons/application/__tests__/ApplicationTypes.ts

```diff
@@ -16,6 +16,7 @@ describe('getLanguageConfig', () => {
   });
 
   test('throws an error for an invalid chapter/variant combination', () => {
+    // @ts-ignore: error is expected for an invalid chapter
```

Comment:
This was added because the TypeScript checker was complaining about the invalid chapter in my editor, so I added this to ignore the error.

Reply:
I don't think is the right approach – it rightly complains because 5 is not a valid chapter. Using `@ts-ignore` alters compile behaviour: we want it to complain instead of silencing it.

Perhaps a typecast would be better? Using `as Chapter` or `as any` (former is preferred) – that way, we tell TS that "yes I know this is invalid, but trust me bro for this one, treat it like it's valid" instead of just ignoring it.